include/ts/apidefs.h.in

```diff
@@ -167,6 +167,15 @@ typedef struct {
   size_t data_size; ///< Amount of message data.
 } TSPluginMsg;
 
+/**
+    A struct to avoid heap allocations in TSMimeHdrField* APIs
+  */
+typedef struct {
+  uint8_t _d[24]; // private
```

Comment:
I think this should be `uint64_t _d[3];` to avoid any possible alignment issues.